### DISCUSSION TITLE
Forward messages to non-Zenoh MQTT broker?

### DISCUSSION THREAD

**Discussion by shane-nichols** [eclipse-zenoh/roadmap#154] [Q&A]
I am currently switching from a Mosquitto to a zenoh subsystem for performance reasons, but still need to send/receive messages on certain topics to/from a non Zenoh broker to communicate with the larger system. Is that currently possibly or a possible feature?

**Comment by Mallets** [comment]
Hello! Have you already looked into [zenoh-plugin-mqtt](https://github.com/eclipse-zenoh/zenoh-plugin-mqtt)? Would that address your needs?
